### ISSUE TITLE
Collapse accessibility : toggle misses aria

### ISSUE THREAD

**Issue by ygatesoupe** [ng-bootstrap/ng-bootstrap#1113]
Collapse toggle should have its own aria-expanded and aria-controls linked to the collapse panel.

This has been corrected for accordion component in ed5d4ca

[Bootstrap doc](http://v4-alpha.getbootstrap.com/components/collapse/#example)

**Comment by pkozlowski-opensource** [comment]
`aria-expanded` is already in place: https://github.com/ng-bootstrap/ng-bootstrap/blob/97f116ff6de4b1acaf1b0a508f7aa461bc59a349/src/collapse/collapse.ts#L9

When it comes to `aria-controls` we can't add it as part of the collapse widget as the controlling button is not part of the widget. `aria-controls` needs to come from the user-land (we could update docs to reflect this).

**Comment by ygatesoupe** [comment]
Hi,

I meant `aria-expended`attribute for the button as well. Check out native bootstrap [implementation](http://v4-alpha.getbootstrap.com/components/collapse/#example).

However I wouldn't know how to implement this within a single component, so better leave it to user side.
